src/components/common/TmSessionExtension.vue

```diff
@@ -68,20 +79,45 @@ export default {
     getAddressesFromExtension() {
       this.$store.dispatch("getAddressesFromExtension")
     },
+    getSignInNetwork(account) {
+      // defaulting to cosmos-hub-mainnet
+      const accountNetwork = account.network || "cosmos-hub-mainnet"
+      if (this.testnet) {
+        return (
+          this.networks
+            .filter(({ testnet }) => testnet)
+            .find(({ slug }) => accountNetwork.startsWith(slug)) ||
```

Comment:
🤮 but ok for now

Reply:
😿

Reply:
I mean, I also tried `.find(({ slug, testnet }) => accountNetwork.startsWith(slug) && testnet)`, but that somehow didn't work 🤷‍♀

Reply:
🤮 is much worse than 🤢 fabo

🌈 is the lunie way 🌈